tests/Unit/IO/Test_H5.cpp

```diff
@@ -447,10 +447,12 @@ SPECTRE_TEST_CASE("Unit.IO.H5.DatRead", "[Unit][IO][H5]") {
   }
 }
 
+// Check that we can insert and open subfiles at the '/' level
 SPECTRE_TEST_CASE("Unit.IO.H5.check_if_object_exists", "[Unit][IO][H5]") {
   const std::string h5_file_name("Unit.IO.H5.ObjectCheck.h5");
   h5::H5File<h5::AccessType::ReadWrite> my_file(h5_file_name);
   auto& error_file = my_file.insert<h5::Header>("/");
+  const auto& header = error_file.get_header();
```

Comment:
? move to other commit

Reply:
Correct me if I'm wrong but I'm fairly sure that `error_file.get_header()` requires opening `error_file`, which is the subfile.